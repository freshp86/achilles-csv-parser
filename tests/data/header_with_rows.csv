foo,bar,baz
,,
"","",""
,2,
,,3
,2,3
1,2,3
hello,world,csv
"hello","world","csv"
hello  ,  world,  csv  
"hello  ","  world","  csv  "
"hello ""world""",2,3
"hello, ""world""",",",3
"hello
""world""",2,3
καλημέρα,Ελλάδα,μας
