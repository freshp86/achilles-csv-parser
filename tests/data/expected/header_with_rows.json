[
  {
    "foo": "",
    "bar": "",
    "baz": ""
  },
  {
    "foo": "",
    "bar": "",
    "baz": ""
  },
  {
    "foo": "",
    "bar": "2",
    "baz": ""
  },
  {
    "foo": "",
    "bar": "",
    "baz": "3"
  },
  {
    "foo": "",
    "bar": "2",
    "baz": "3"
  },
  {
    "foo": "1",
    "bar": "2",
    "baz": "3"
  },
  {
    "foo": "hello",
    "bar": "world",
    "baz": "csv"
  },
  {
    "foo": "hello",
    "bar": "world",
    "baz": "csv"
  },
  {
    "foo": "hello  ",
    "bar": "  world",
    "baz": "  csv  "
  },
  {
    "foo": "hello  ",
    "bar": "  world",
    "baz": "  csv  "
  },
  {
    "foo": "hello \"world\"",
    "bar": "2",
    "baz": "3"
  },
  {
    "foo": "hello, \"world\"",
    "bar": ",",
    "baz": "3"
  },
  {
    "foo": "hello; \"world\"",
    "bar": ";",
    "baz": "3"
  },
  {
    "foo": "hello\n\"world\"",
    "bar": "2",
    "baz": "3"
  },
  {
    "foo": "καλημέρα",
    "bar": "Ελλάδα",
    "baz": "μας"
  }
]